cro-clib/chain.h

```diff
@@ -135,3 +176,46 @@ CroResult cro_get_printed_address(CroAddressPtr address_ptr,
  * # Safety
  */
 CroResult cro_restore_hdwallet(const char *mnemonics_string, CroHDWalletPtr *wallet_out);
+
+/**
+ * add txin
+ * txid_string: 64 length hex-char , 32 bytes
+ * addr_string: transfer address
+ * cro_string: cro unit , example 0.0001
+ * # Safety
+ */
+CroResult cro_tx_add_txin(CroTxPtr tx_ptr,
```

Comment:
I guess other transaction types (deposit/unbund/withdraw etc.) are postponed for later pr?

Reply:
yes, after this pr is reviewed